src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -837,31 +837,31 @@ private Game GetNextLoserGame(Game finishedGame, List<Game> games)
             return nextGame;
         }
 
-        private int GetNextGameNumber(Game finishedGame, List<Game> games)
+        private int GetNextGameNumber(Game finishedGame, ICollection<Game> games)
         {
             int numberOfRounds = GetNumberOfRounds(finishedGame, games);
 
             return ((finishedGame.GameNumber + 1) / 2)
                 + Convert.ToInt32(Math.Pow(2, numberOfRounds - 1));
         }
 
-        private bool IsSemiFinalGame(Game finishedGame, List<Game> games)
+        private bool IsSemiFinalGame(Game finishedGame, ICollection<Game> games)
         {
             int numberOfRounds = GetNumberOfRounds(finishedGame, games);
             List<Game> gamesInCurrentRound = games.Where(g => g.Round == finishedGame.Round).ToList();
 
             return finishedGame.Round == numberOfRounds - 1;
         }
 
-        private int GetNumberOfRounds(Game finishedGame, List<Game> games)
+        private int GetNumberOfRounds(Game finishedGame, ICollection<Game> games)
         {
             List<Game> gamesInCurrntRound = games.Where(g => g.Round == finishedGame.Round).ToList();
 
             return Convert.ToInt32(Math.Log(gamesInCurrntRound.Count(), 2))
                 + finishedGame.Round;
         }
 
-        private bool IsGameInLastRound(Game finishedGame, List<Game> games)
+        private bool IsGameInLastRound(Game finishedGame, ICollection<Game> games)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Collections.Generic.IEnumerable<VolleyManagement.Domain.GamesAggregate.Game>' instead of 'System.Collections.Generic.ICollection<VolleyManagement.Domain.GamesAggregate.Game>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)

Reply:
?